playwright-test/atb.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { test, expect, mockAtb } from "./harness";
```

Comment:
Shouldn't the legacy atb integration tests be removed?

Reply:
Yes. Just left here for comparison.